testing/testcafe/tests/editors/validationMessage.ts

```diff
@@ -20,3 +27,55 @@ test('ValidationMessage integrated in editor should not raise any errors when it
     .click(checkBox1.element)
     .expect(true).ok();
 });
+
+fixture`ValidationMessage_TextBox`
+  .page(url(__dirname, '../container.html'));
+
+test('Validation Message position should be correct after change visibility of parent container', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await t
+    .click(Selector(`.${TEXTEDITOR_INPUT_CLASS}`))
+    .pressKey('backspace')
+    .pressKey('enter')
+    .pressKey('tab');
+
+  await t
+    .expect(await takeScreenshot('Textbox_validation_message.png'))
+    .ok()
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+
+  await ClientFunction(() => {
+    (document.querySelector('#container') as HTMLElement).setAttribute('hidden', 'true');
+  })();
+
+  await ClientFunction(() => {
+    document.querySelector('#container')?.removeAttribute('hidden');
+  })();
+
+  await t
+    .expect(await takeScreenshot('Textbox_validation_message.png'))
+    .ok()
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async (t) => {
+  await t.resizeWindow(300, 200);
```

Comment:
why do we need this window resize?

Reply:
I need to do screenshot which contains the Texteditor and the ValidationMessage simultaneously. There is no container element that include both of them. So, I get minimal screenshot dimensions for my purpose through resize windows.